unified-codes/tools/scripts/dgraph/migration/v1/import.js

```diff
@@ -0,0 +1,161 @@
+const fs = require('fs').promises;
+const excel = require('exceljs');
+const jsonFormat = require('json-format');
+
+const inputFilename = process.argv[2] || './products.xlsx';
+const outputFilename = process.argv[3] || './import.json';
+const items = [];
+
+const properties = [
+  'code',
+  'title',
+  'type',
+  'ddd',
+  'rxnav',
+  'eml',
+  'atc',
+  'unspsc',
+  'nzulm',
+  'form_category',
+  'form',
+  'code',
+  'title',
+  'strength',
+  'ddd',
+  'rxnav',
+  'eml',
+  'atc',
+  'unspsc',
+  'nzulm',
+];
```

Comment:
See previous comment... something like this!

![yes_2](https://user-images.githubusercontent.com/43223637/99205672-1b795b00-281e-11eb-8330-84aa0581d388.gif)